src/Our.Umbraco.StackedContent/Web/Helpers/ViewHelper.cs

```diff
@@ -1,15 +1,36 @@
 ﻿using System.IO;
+using System.Linq;
 using System.Web;
 using System.Web.Mvc;
 using System.Web.Routing;
+using Umbraco.Core.Logging;
 
 namespace Our.Umbraco.StackedContent.Web.Helpers
 {
-    internal static class ViewHelper
+    public static class ViewHelper
     {
-        class DummyController : Controller { }
+        private class DummyController : Controller { }
 
-        public static string RenderPartial(string partialName, object model, string[] viewLocations)
+        private static readonly RazorViewEngine ViewEngine = new RazorViewEngine
+        {
+            PartialViewLocationFormats = new[]
+            {
+                "~/Views/Partials/Stack/{0}.cshtml",
+                "~/Views/Partials/Stack/Default.cshtml"
+            }
+        };
+
+        public static void AddViewLocationFormats(params string[] viewLocationFormats)
+        {
+            var newFormats = ViewEngine
+                .PartialViewLocationFormats
+                .Union(viewLocationFormats)
+                .ToArray();
+
+            ViewEngine.PartialViewLocationFormats = newFormats;
```

Comment:
This ties in with my previous comment, with `ViewEngine` being a static field - @sniffdk where would you be calling `AddViewLocationFormats` from? I'd expect, at app start-up, but if this is called elsewhere, then it could be called multiple times and the same `viewLocationFormats` would be repeatedly appended.

Wondering if it needs a check to see if the locations already exist in `PartialViewLocationFormats`?
or maybe that is overkill?

Reply:
We would indeed normally call them from startup code. This code purposefully use the .Union syntax to avoid duplicates in the list. Granted, it's case sensitive, but if you run into trouble there, that might not be your biggest problem? 😅

Reply:
Could always `.ToLower()` just to be sure :)

Reply:
You know, I didn't know that `.Union` handled distincts, I thought it appended it.
Everyday's a school day! 😀

Reply:
To be honest though, accidentally ending up with two identical locations with different casing wouldn't really do anything, it's just a string array.